addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -697,4 +701,10 @@
     <i t-if="slide.slide_type == 'quiz'" t-att-class="'fa fa-question-circle %s' % icon_class"></i>
 </template>
 
+<template name="Display Promoted Slide" id="display_promoted_slide" inherit_id="website_slides.course_promoted_slide" active="True" customize_show="True">
+    <xpath expr="//div[hasclass('promoted-slide')]" position="attributes">
+        <attribute name="class">container py-1 mb-2 promoted-slide</attribute>
```

Comment:
I would wrap the base template in another div that just has `o_wslides_promoted_slides d-none`

And then you re-write to `o_wslides_promoted_slides`

That way you don't have to specify `container py-1 mb-2` twice.